vtex/utils/transform.ts

```diff
@@ -1120,6 +1121,15 @@ function toHoursSpecification(hours: Hours): OpeningHoursSpecification {
   };
 }
 
+function toHolidayHoursSpecification(holiday: PickupHolidays) {
+  return {
+    "@type": "PickupHolidays",
+    date: holiday.date,
+    hourBegin: holiday.hourBegin,
+    hourEnd: holiday.hourEnd,
+  };
+}
+
```

Comment:
Rewrite this function to return `OpeningHoursSpecification` instead of `PickupHolidays`

```suggestion
function toHolidayHoursSpecification(holiday: PickupHolidays): OpeningHoursSpecification {
  return {
    "@type": "OpeningHoursSpecification",
    // rewrite using OpeningHoursSpecification properties
  };
}
```